lib/pages/settings_dashboard/settings/settings.dart

```diff
@@ -257,6 +262,30 @@ class SettingsController extends State<Settings> with ConnectPageMixin {
         PlatformInfos.showAboutDialogFullScreen();
       case SettingEnum.logout:
         logoutAction();
+        break;
+      case SettingEnum.deleteAccount:
+        if (await showConfirmAlertDialog(
+              useRootNavigator: false,
+              context: context,
+              responsiveUtils: _responsiveUtils,
+              title: L10n.of(context)!.deleteAccount,
+              titleColor: Theme.of(context).colorScheme.error,
+              message: L10n.of(context)!.deleteAccountMessage,
+              okLabel: L10n.of(context)!.next,
+              cancelLabel: L10n.of(context)!.cancel,
+              maxLinesMessage: 5,
+            ) ==
+            ConfirmResult.cancel) {
+          return;
+        }
+
+        final emailUri = Uri.parse(
+          'mailto:software@linagora.com?subject=${Uri.encodeComponent(generateEmailSubject)}',
+        );
+        if (await canLaunchUrl(emailUri)) {
```

Comment:
please test case no email was logged in

Reply:
It's test for this case
https://github.com/user-attachments/assets/60ed4877-0e09-4ced-a482-0ded9e84f8c9